shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeA.js

```diff
@@ -4,44 +4,48 @@ const {
   SERVICE_STAMP_OPTIONS,
   VALIDATION_ERROR_MESSAGES,
 } = require('./CourtIssuedDocumentConstants');
+const {
+  JoiValidationConstants,
+} = require('../../../utilities/JoiValidationConstants');
 const {
   joiValidationDecorator,
+  validEntityDecorator,
 } = require('../../../utilities/JoiValidationDecorator');
 const { replaceBracketed } = require('../../utilities/replaceBracketed');
 
 /**
  * @param {object} rawProps the raw document data
  * @constructor
  */
-function CourtIssuedDocumentTypeA(rawProps) {
+function CourtIssuedDocumentTypeA() {}
+CourtIssuedDocumentTypeA.prototype.init = function init(rawProps) {
   this.attachments = rawProps.attachments;
   this.documentTitle = rawProps.documentTitle;
   this.documentType = rawProps.documentType;
   this.freeText = rawProps.freeText;
   this.serviceStamp = rawProps.serviceStamp;
-}
+};
 
 CourtIssuedDocumentTypeA.prototype.getDocumentTitle = function () {
   return replaceBracketed(this.documentTitle, this.freeText);
 };
 
 CourtIssuedDocumentTypeA.schema = {
   attachments: joi.boolean().required(),
-  documentTitle: joi.string().optional(),
```

Comment:
I wonder if we could consolidate the common attributes in these different CourtIssuedDocumentType{letter} entity files, so you can see the difference between them at a glance.

Reply:
`attachments`, `documentType`, and `documentTitle` are about the only fields that are shared across these Entities, and some of them are handled differently in the constructor (for example, `documentTitle` is usually composed of other fields on the Entity). I'll add a DevEx card to Trello to discuss this.

Reply:
https://trello.com/c/eWIAmFqM/776-refactor-common-external-document-type-properties